documentation/staging/content/managing-domains/accessing-the-domain/domain-events.md

```diff
@@ -52,10 +55,17 @@ The operator generates these event types in a domain namespace, which indicate t
  * `Completed`:  The domain resource is complete because all of the following are true: there is no failure detected, there are no pending server shutdowns, and all servers expected to be running are ready and at their target image, auxiliary images, restart version, and introspect version.all servers that are supposed to be started are up running.
  * `Unavailable`: The domain resource is unavailable, which means the domain does not have a sufficient number of servers active.
  * `Incomplete`: The domain resource is incomplete for one or more of the following reasons: there are failures detected, there are pending server shutdowns, or not all servers expected to be running are ready and at their target image, auxiliary images, restart version, and introspect version.
- * `FailureRessolved`: The [failure condition]({{< relref "/managing-domains/accessing-the-domain/status-conditions#types-of-domain-conditions" >}}) that the domain was in has been resolved.
+ * `FailureResolved`: The [failure condition]({{< relref "/managing-domains/accessing-the-domain/status-conditions#types-of-domain-conditions" >}}) that the domain was in has been resolved.
  * `RollStarting`:  The operator has detected domain resource or Model in Image model
     updates that require it to perform a rolling restart of the domain.
  * `RollCompleted`:  The operator has successfully completed a rolling restart of a domain.
+ * `ClusterCreated`: A new cluster resource is created.
+ * `ClusterChanged`: A change has been made to an existing cluster resource.
+ * `ClusterDeleted`: An existing cluster resource has been deleted.
+ * `ClusterAvailable`: An existing cluster is available, which means a sufficient number of its servers have reached the ready state.
+ * `ClusterCompleted`: The cluster is complete because all of the following are true: there is no failure detected, there are no pending server shutdowns, and all servers expected to be running are ready and at their target image, auxiliary images, restart version, and introspect version.
+ * `ClusterIncomplete`: The cluster is incomplete for one or more of the following reasons: there are failures detected, there are pending server shutdowns, or not all servers expected to be running are ready and at their target image, auxiliary images, restart version, and introspect version.
+ * `ClusterUnvailable`: The cluster is unavailable because an insufficient number of its servers that are expected to be running are ready.
```

Comment:
`ClusterAvailable/Completed/Incomplete/Unavailable` all could use a link to the new conditions doc. Something like: "For details, see the corresponding \[condition\]\(link\)."